content/tutorial/02-sveltekit/02-routing/04-loading-layout-data/README.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Loading Layout Data
+---
+
+Layouts can provide common UI across pages. They can also provide common data across pages.
+
+Providing this data is similar to the `load` function in `+page.js`, only that we place it inside `+layout.js` - notice the symmetry?
+
+Like in `+page.js`, this `load` function runs on the server when the user visits your page initially, and on the client for all further navigations - it's isomorphic. We call it a "shared `load` function". To harmonize the differences between client and server, use the `fetch` function passed to `load` to load data. The return type of the `load` function needs to be an object at the top level, below that everything is allowed. If we return a promise, it's awaited. The result is passed to the `data` prop inside `+layout.svelte`, so we can access it through `export let data`.
```

Comment:
I did it on purpose in case anyone comes from Google and hasn't done the previous chapter - but yeah for someone doing it linearly it's duplicated. Not sure.